Cargo.toml

```diff
@@ -50,6 +50,9 @@ tracing-futures = { version = "0.2.0", default-features = false, features = ["st
 tracing-subscriber = { version = "0.3.0", default-features = false, features = ["env-filter", "fmt", "ansi", "time", "local-time"] }
 url = "2"
 windows-sys = { version = "0.59", features = ["Win32_Foundation", "Win32_System_IO", "Win32_Networking_WinSock"] }
+getrandom = { version = "0.2", default-features = false }
```

Comment:
Please keep the dependencies in alphabetic order. Also, it doesn't look like `rustls-pki-types` actually gets used here? And also please simplify `{ version = "x" }` to just `"x"`.